src/applications/appeals/10182/pages/boardReview.js

```diff
@@ -1,34 +1,54 @@
+import {
+  radioSchema,
+  radioUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
 import {
   boardReviewTitle,
-  boardReviewContent,
+  boardReviewLabels,
+  boardReviewDescriptions,
   boardReviewErrorMessage,
   BoardReviewReviewField,
 } from '../content/boardReview';
 
 const boardReview = {
   uiSchema: {
     boardReviewOption: {
-      'ui:title': boardReviewTitle,
-      'ui:reviewField': BoardReviewReviewField,
-      'ui:widget': 'radio',
-      'ui:options': {
-        labels: boardReviewContent,
+      ...radioUI({
+        title: boardReviewTitle,
+        labelHeaderLevel: '3',
+        labels: boardReviewLabels,
+        descriptions: boardReviewDescriptions,
         enableAnalytics: true,
-      },
-      'ui:errorMessages': {
-        required: boardReviewErrorMessage,
-      },
+        errorMessages: {
+          required: boardReviewErrorMessage,
+        },
+      }),
+      'ui:reviewField': BoardReviewReviewField,
```

Comment:
Curious why the others don't have the `ui:` syntax anymore while `reviewField` does.

Reply:
The `radioUI` function builds the uiSchema so the above are all props for that function - it in turn adds the `ui:` in front.